docs/imviz/displayimages.rst

```diff
@@ -0,0 +1,92 @@
+.. _imviz-display-images:
+
+*****************
+Displaying Images
+*****************
+
+Imviz uses image viewers to visualize data from supported formats.
+
+Selecting Data Set
+==================
+
+.. seealso::
+
+    :ref:`Selecting Data Set<cubeviz-selecting-data>`
+        Documentation on selecting data sets in the Jdaviz viewers.
+
+
+Adding New Viewers
+==================
+
+In the toolbar towards the top of the UI, there is a :guilabel:`picture with a plus` icon
+
+.. image:: img/picture_with_plus.png
+
+that when clicked will add new viewers to the application. You can then select from the data
+that has been loaded into the application to be visualized in these additional viewers.
+You can then utilize some of the Imviz-specific features, like Linked Pan/Zoom.
+
+Defining Spatial Regions
+========================
+
+Spatial regions can be created the same way as in Cubeviz.
+
+.. seealso::
+
+    :ref:`Defining Spatial Regions <spatial-regions>`
+        Documentation on defining spatial regions in an image viewer.
+
+Blinking
+========
+
+Blinking is an Imviz-specific functionality that allows a user to quickly switch
+between viewing two or more images, as long as they are linked. This can be done
```

Comment:
I think this is good enough, unless you want a new section on Imviz linking?
```suggestion
between viewing two or more images, as long as they are linked
(see :ref:`imviz_pan_zoom` for more on linking behavior). This can be done
```